addons/project_hr_expense/__manifest__.py

```diff
@@ -5,7 +5,7 @@
 {
     'name': 'Project Expenses',
     'version': '1.0',
-    'category': 'Services/expenses',
+    'category': 'Human Resources/Expenses',
```

Comment:
Seems opinionated to me

Reply:
The goal of the bridge is to be able to take the expenses into account in the project profitability. This should remain `Services` to me.

Reply:
It is a module created for Services apps (more precisely for Project app), so I would keep this module in Services too.

Reply:
@IT-Ideas @xavierbol ,
updated the pr by reverting back to the original category and just capitalized e of Expenses